backend/src/main/java/com/bakdata/conquery/apiv1/query/TableExportQuery.java

```diff
@@ -195,27 +203,31 @@ private Map<SecondaryIdDescription, Integer> calculateSecondaryIdPositions(Atomi
 		return secondaryIdPositions;
 	}
 
-	private static List<ResultInfo> createResultInfos(Map<SecondaryIdDescription, Integer> secondaryIdPositions, Map<Column, Integer> positions, @NotEmpty @Valid List<CQConcept> tables, boolean rawConceptColumns) {
+	private List<ResultInfo> createResultInfos(Map<SecondaryIdDescription, Integer> secondaryIdPositions) {
 
 		final int size = positions.values().stream().mapToInt(i -> i).max().getAsInt() + 1;
 
 		final ResultInfo[] infos = new ResultInfo[size];
 
-		infos[0] = ConqueryConstants.DATES_INFO;
+		final Set<SecondaryIdDescription> grouping = previewConfig.getGrouping();
+
+		infos[0] = ConqueryConstants.DATES_INFO_HISTORY;
 		infos[1] = ConqueryConstants.SOURCE_INFO;
 
 		for (Map.Entry<SecondaryIdDescription, Integer> e : secondaryIdPositions.entrySet()) {
-			SecondaryIdDescription desc = e.getKey();
-			Integer pos = e.getValue();
-
-			infos[pos] = new SimpleResultInfo(
-					desc.getLabel(),
-					ResultType.StringT.INSTANCE,
-					Set.of(
-							new SemanticType.SecondaryIdT(desc),
-							new SemanticType.DescriptionT(desc.getDescription())
-					)
-			);
+			final SecondaryIdDescription desc = e.getKey();
+			final Integer pos = e.getValue();
+
+			final Set<SemanticType> semantics = new HashSet<>();
+
+			semantics.add(new SemanticType.SecondaryIdT(desc));
+			semantics.add(new SemanticType.DescriptionT(desc.getDescription()));
+
+			if (grouping.contains(desc)){
+				semantics.add(new SemanticType.HiddenT());
+			}
```

Comment:
Sollte das kein Fehler sein, dann bitte erklären, warum hier `HiddenT` hinzugefügt wird, wenn es in `grouping` ist

Reply:
weiter unten ist noch ein `Hidden` block, ich glaube das ist ein Fehler hier

Reply:
Das war ein Fehler